openc3/lib/openc3/packets/packet_config.rb

```diff
@@ -469,6 +473,13 @@ def process_current_packet(parser, keyword, params)
         @current_packet.hidden = true
         @current_packet.disabled = true
 
+      when 'VIRTUAL'
+        usage = "#{keyword}"
+        parser.verify_num_parameters(0, 0, usage)
+        @current_packet.hidden = true
+        @current_packet.disabled = true
+        @current_packet.virtual = true
```

Comment:
What is a VIRTUAL packet and how is it going to be used?

Reply:
VIRTUAL packs aren't involved in identification.  So it provides a way to declare embedded structures that can be used within packets.   Effectively packets in packets.